WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/DummyActivityTypesProvider.kt

```diff
@@ -0,0 +1,30 @@
+package org.wordpress.android.ui.activitylog.list.filter
+
+import kotlinx.coroutines.delay
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * This class will be deleted as soon as we implement fetchAvailableActivityTypes endpoint in ActivityLogStore.
+ */
+@Singleton
+class DummyActivityTypesProvider @Inject constructor() {
```

Comment:
Love (❤️ ): Loving it, nice way of implementing stub functionality and proceed forward with the feature!